src/components/HeatmapCell.js

```diff
@@ -23,11 +23,28 @@ const getcolor = (value) => {
   return color;
 }
 
-const HeatmapCell = ({value, isHighlighted}) => {
+const HeatmapCell = ({value, isDimmed, isHighlighted}) => {
```

Comment:
Can you combine isDimmed and isHighlighted into one variable? Since they will never be both true at the same time.

It could be called display or whatever you want and it could equal "dimmed", "highlighted", "normal", more...